conf/nginx/st2.conf

```diff
@@ -43,7 +43,14 @@ server {
   add_header              Front-End-Https on;
   add_header              X-Content-Type-Options nosniff;
 
+  location @apiError {
+    add_header Content-Type application/json;
+    return 200 '{ "faultstring": "Nginx is unable to reach st2api. Make sure service is running." }';
```

Comment:
200 is of course an awful return code here, but nginx doesn't allow to change headers on 4xx and 5xx responses up until 1.7.5 version and ubuntu's current one is 1.4.6. We'll fix that in the next commit.